examples/with-meilisearch/README.md

```diff
@@ -0,0 +1,24 @@
+# Next.js with Meilisearch
+
+This starter configures Meilisearch and InstantSearch for a Next.js application.
+
+## Deploy your own
+
+Deploy this project on Vercel:
+
+[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/clone?repository-url=https://github.com/vercel/next.js/tree/canary/examples/with-meilisearch&project-name=with-meilisearch&repository-name=with-meilisearch)
+
+## How to use
+
+1. Create a [Meilisearch Cloud project](https://cloud.meilisearch.com)
+1. Run `npx create-next-app --example with-meilisearch my-app` to create a Next.js app using the Meilisearch starter kit
+1. Navigate to your application directory, eg. `cd my-app`
```

Comment:
Please follow the structure of the other example READMEs: https://github.com/vercel/next.js/blob/canary/contributing/examples/adding-examples.md

Reply:
I've updated it to match the new guidelines for the "How to use" section.

I still had to include a notice about the user needing an account/credentials.

l hope that's ok? let me know if it isn't :)